src/mutation.cpp

```diff
@@ -1738,6 +1747,22 @@ void Character::mutation_spend_resources( const trait_id &mut )
         if( mdata.fatigue ) {
             mod_fatigue( cost );
         }
+        if( mdata.stamina ) {
+            mod_stamina( -cost ); // flipped, because it should be consuming stamina not adding to it
+        }
+        if( mdata.mana ) {
+            magic->mod_mana( *this, -cost ); // flipped, because it should be consuming mana not adding to it
+        }
+        if( mdata.health ) {
+            mod_healthy( -cost ); // flipped, because it should be consuming health not adding to it
+        }
+        if( mdata.pain ) {
+            mod_pain( cost );
+        }
+        if( mdata.bionic ) {
+            mod_power_level( units::from_kilojoule(
+                                 -cost ) );  // flipped, because it should be consuming bionic power not adding to it
```

Comment:
```suggestion
            // flipped, because it should be consuming bionic power not adding to it
            mod_power_level( units::from_kilojoule( -cost ) );
```
Move comment to its own line so that astyle doesn't stupidly split the line that should, reasonable, stay unsplit